.gitignore

```diff
@@ -17,6 +17,10 @@ data
 vector-db
 vendor
 
+# ignore credential files
+bam_api_key.txt
+openai_api_key.txt
+
```

Comment:
`*_api_key.txt` to establish a pattern and leave a comment to that effect in olsconfig.yaml?

Reply:
i'll update the git ignore rule, but ultimately olsconfig.yaml is going to be machine generated by our operator, so i don't think we need to define strong patterns for naming conventions there.